### ISSUE TITLE
ResNet-20 accuracy on CIFAR-10

### ISSUE THREAD

**Issue by skvenka5** [AngusG/bn-advex-zhang-fixup#1]
Hey, I am trying to reproduce resnet20 fixup results given in "fixup-vs-batchnorm-lab-clean-eps-converted-to.pdf". But, after 200 epochs I am getting only ~35% accuracy on CIFAR-10. I ran the model with default hyperparameters using the following command
`CUDA_VISIBLE_DEVICES=0 python cifar_train.py -a fixup_resnet20`
What could be the issue?

**Comment by AngusG** [comment]
Hi, sorry I didn't notice this issue until now. 

Are you still stuck with this? If so can you paste the output of the first few training epochs here? And maybe also your pytorch version?

I just ran `python cifar_train.py --dataroot /path/to/my/dataset/ --logdir /log/path/ -a fixup_resnet20 --sess skvenka5` and have ~80% test accuracy after only 15 epochs. 

The best accuracy is 91.66% @ epoch 191, compatible with the "Fixup" data series of figure you mention.
